src/main/java/model/LottoNumber.java

```diff
@@ -0,0 +1,30 @@
+package model;
+
+public class LottoNumber {
+    public static final int LOTTO_MIN_NUMBER = 1;
+    public static final int LOTTO_MAX_NUMBER = 45;
+    private static final String LOTTO_NUMBERS_MUST_BETWEEN_LOTTO_MIN_NUMBER_AND_LOTTO_MAX_NUMBER = "로또 번호는 1부터 45 사이의 숫자로 구성되어야 합니다.";
+
+    private final int number;
+
+    public LottoNumber() {
+        int number = createNumber();
+        validateNumber(number);
+        this.number = number;
+    }
+
+    public static void validateNumber(int number) {
+        if (LOTTO_MIN_NUMBER <= number && number <= LOTTO_MAX_NUMBER) {
+            return;
+        }
+        throw new IllegalArgumentException(LOTTO_NUMBERS_MUST_BETWEEN_LOTTO_MIN_NUMBER_AND_LOTTO_MAX_NUMBER);
```

Comment:
```suggestion
        if (LOTTO_MIN_NUMBER > number && number > LOTTO_MAX_NUMBER) {
                  throw new IllegalArgumentException(LOTTO_NUMBERS_MUST_BETWEEN_LOTTO_MIN_NUMBER_AND_LOTTO_MAX_NUMBER);
        }
```

위와 같이 변경하는 것이 예외의 의미도 더 명확해지고, 또한 코드의 줄 수도 줄일 수 있을 것 같습니다.

Reply:
좋은 지적 감사합니다. 반영하도록 하겠습니다.